src/framework/application.js

```diff
@@ -1667,6 +1681,27 @@ class Application extends EventHandler {
         }
     }
 
+    /**
+     * @function
+     * @private
+     * @name pc.Application#setAreaLightData
+     * @description Sets the area light LUT asset for this app.
+     * @param {pc.Asset} asset - Asset of type `binary` to be set.
+     */
+    setAreaLightData(asset) {
```

Comment:
renaming this to setAreaLightLuts would match renderer._uploadAreaLightLuts better too.

Reply:
agree - and done